nav2_system_tests/src/localization/test_localization_launch.py

```diff
@@ -115,7 +115,13 @@ def main(argv=sys.argv[1:]):
     lts.add_test_action(ld, test1_action)
     ls = LaunchService(argv=argv)
     ls.include_launch_description(ld)
-    return lts.run(ls)
+    return_code = lts.run(ls)
```

Comment:
I think you may have forgotten to add in the kill command here

Reply:
waiting for the other pr to merge